package.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "@types/mediawiki",
+	"version": "0.1.0",
+	"description": "TypeScript definitions of MediaWiki interfaces",
+	"scripts": {
+		"build:types": "tsc",
+		"build": "npm-run-all build:*",
+		"test:lint": "eslint --cache --max-warnings 0 **/*.ts",
```

Comment:
Ultra nitpick (feel free to ignore): I'm not sure but I think this can just be `--ext .ts` or maybe `--ext .ts .` to avoid possible issues with shell wildcards.

I'm not sure what Windows shells will expect but I think macOS and Linux will attempt to glob unquoted / unescaped wildcards like the `build:*` in `"build": "npm-run-all build:*",`. This could be replaced with `"build": "npm-run-all build:\*",` to avoid the very rare situation where someone has a file named `build:foo` in their directory.

Reply:
Good point about specifying extension using the dedicated parameter. Swapped.

I am tending to leave the "wildcard" it as it is, and have the escaping added when people see it is preferred,